examples/docker-compose.yaml

```diff
@@ -0,0 +1,5 @@
+version: '3.4'
```

Comment:
Question: maybe move this to the repo root, and then the example can be something like: 

```
docker-compose run xk6-browser run - <examples/browser_on.js
```

WDYT?

Reply:
I don't have a very strong opinion on this. I think leaving it here for now works, and can move it later if we need.

Reply:
How would we run it if we didn't have docker-compose?

Reply:
>How would we run a script if we didn't have docker-compose?

Well, by using docker... But are there any issues with the `docker-compose` many other projects use, it's a kind of standard way of quickly trying things :shrug:

Reply:
I think there's much value using Docker Compose when we only have a single service, but if you guys think we should keep it, I'm not strongly against it.

Reply:
👍 for the docker-compose. In my opinion, it's simpler to run a single command (`docker-compose`) rather than running it through `docker` (build + run steps).

Reply:
Yeah, at least till the point when CI gets the builds & pushes them to the docker hub, then probably nothing else apart from the will be needed:

```
docker run -i grafana/xk6-browser run - <examples/fillform.js
```